frigate/prometheus_exporter.py

```diff
@@ -0,0 +1,219 @@
+import json
```

Comment:
I think it might be good to go ahead and create a `stats` folder and include the stats.py as well as this file (maybe with different names) in that folder.

Reply:
I think this makes sense

Reply:
Any Name for stats class? We will get 
![image](https://user-images.githubusercontent.com/6330506/221708951-08794092-a36f-4df9-b484-bca2565f191c.png)

Reply:
My personal vote would be to just call it frigate (so it would be `frigate.stats.frigate` but maybe there is a better name

Reply:
@blakeblackshear  if you have any opinions please share. I'm not a daily python user, so not familiar with conventions.

Reply:
How about a root folder called monitoring, with two files, "stats.py" and "prometheus_exporter.py" ?

Reply:
When applications use the Prometheus libraries themselves, the proper term would be instrumentation, not exporter.

Reply:
@mitchross `stats.py` and `prometheus_metrics.py` seem fine.